app/assets/javascripts/models/data/preferences.js.coffee

```diff
@@ -51,11 +57,13 @@ ns.Preferences = do (ko
     fromJson: (jsonObj) =>
       return unless jsonObj?
       @showTour(jsonObj.show_tour != 'false')
+      @doNotShowTourAgain(jsonObj.doNotShowTourAgain)
```

Comment:
@mightynimble - done.  All those console entries should show strings after I push here in a bit.